Tasks/VsTest/task.json

```diff
@@ -17,7 +17,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 3
+        "Patch": 4
```

Comment:
>"Patch": 4 [](start = 8, length = 10)

No need to change the patch version, if only legacy test suite is getting modified?